client/components/jetpack/card/jetpack-free-card/style.scss

```diff
@@ -41,6 +50,10 @@
 
 	font-size: 1rem;
 
+	@include breakpoint-deprecated( '>660px' ) {
```

Comment:
Only limit the button width once the screen is wide enough to support multiple columns in the grid.